be/src/exec/CMakeLists.txt

```diff
@@ -205,6 +206,8 @@ set(EXEC_FILES
 if ("${CMAKE_BUILD_TARGET_ARCH}" STREQUAL "x86" OR "${CMAKE_BUILD_TARGET_ARCH}" STREQUAL "x86_64")
     set_source_files_properties(vectorized/json_scanner.cpp PROPERTIES COMPILE_FLAGS -mno-avx)
     set_source_files_properties(vectorized/json_scanner.cpp PROPERTIES COMPILE_FLAGS -mno-avx2)
+    set_source_files_properties(vectorized/json_parser.cpp PROPERTIES COMPILE_FLAGS -mno-avx)
```

Comment:
these flags could be merged into one line

Reply:
updated.